kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

```diff
@@ -758,10 +758,18 @@ private QueryDefinition createQueryDefinition(String target) {
     }
 
     private QueryDefinition createErrorsQueryDefinition() {
+        String queryExpression = "select * from ExecutionErrorInfo where ERROR_ACK = ";
+        // Sybase needs special treatment when it comes to boolean
+        if (TestConfig.getKieServerDataSourceDriverClass().startsWith("com.sybase")) {
```

Comment:
Just an idea, this condition could be extracted to separate method to make it more readable. Something like TestConfig.isSybaseDataSource(). That can be reused in the future for other tests.